handler_error.go

```diff
@@ -227,11 +226,9 @@ func (e *ErrorHandler) HandleError(w http.ResponseWriter, r *http.Request, errMs
 		if prior, ok := r.Header["X-Forwarded-For"]; ok {
 			clientIP = strings.Join(prior, ", ") + ", " + clientIP
 		}
-		ip = clientIP
 	} else {
 		log.WithFields(logrus.Fields{
 			"prefix":      "gateway",
-			"user_ip":     ip,
```

Comment:
We need the IP in there, this might be a dead var, but we can;t losr it from the log message - some users require IP traces for security auditing.